hw12_13_14_15_calendar/service/server/utils.go

```diff
@@ -35,10 +35,11 @@ func ConvertTimeToTimestamp(ntime sql.NullTime) (result *timestamp.Timestamp, er
 // Didn't want to use reflection.
 func ConvertEventToProto(evt repository.Event) (*Event, error) {
 	result := &Event{
-		ID:          evt.ID,
-		UserID:      evt.UserID,
+		ID:          int32(evt.ID),
```

Comment:
https://github.com/protocolbuffers/protobuf/issues/2679
strange "feature", so need to move to `int/int32`